src/extra.ml

```diff
@@ -71,14 +71,21 @@ module Option =
       | (None    , None    ) -> true
       | (Some(e1), Some(e2)) -> eq e1 e2
       | (_       , _       ) -> false
+
+    (** [pp pp_elt oc o] prints on the channel [oc] the element [e] with
+       [pp_elt] if [o = Some e]. *)
+    let pp : 'a pp -> 'a option pp = fun pp_elt oc o ->
+      match o with
+      | None   -> ()
+      | Some e -> pp_elt oc e
   end
 
 module List =
   struct
     include List
 
-    (** [pp pp_e sep oc l] prints the list [l] on the channel [oc] using [sep]
-        as separator, and [pp_e] for printing the elements. *)
+    (** [pp pp_elt sep oc l] prints the list [l] on the channel [oc] using
+       [sep] as separator, and [pp_elt] for printing the elements. *)
```

Comment:
This change has nothing to do with the current PR and it breaks consistency with the rest of the file.

Reply:
Right. But I updated the comment so that it uses the same names as the ones in the code.

Reply:
Well, it does not use the same name as the rest of the file. You should rather have used the same name in your new code!

Reply:
Please, fix the indentation in the comment.